File: webmvc/src/main/java/org/sample/webmvc/configuration/JsonNullableConfiguration.java
package org.sample.webmvc.configuration;

import com.fasterxml.jackson.databind.Module;
import org.openapitools.jackson.nullable.JsonNullableModule;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class JsonNullableConfiguration {
    @Bean
    public Module jsonNullableModule() {
        return new JsonNullableModule();
    }
}

File: webflux-kt/src/main/kotlin/org/sample/webflux/configuration/JsonNullableConfiguration.kt
package org.sample.webflux.configuration

import com.fasterxml.jackson.databind.Module
import org.openapitools.jackson.nullable.JsonNullableModule
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class JsonNullableConfiguration {
    @Bean
    fun jsonNullableModule(): Module {
        return JsonNullableModule()
    }
}
